tools/syz-query-subsystems/query_subsystems.go

```diff
@@ -58,7 +58,10 @@ func main() {
 	if err = osutil.MkdirAll(folder); err != nil {
 		tool.Failf("failed to create %s: %v", folder, err)
 	}
-	commitInfo := determineCommitInfo(*flagKernelRepo)
+	commitInfo, err := determineCommitInfo(*flagKernelRepo)
```

Comment:
Double checking: previously is was commit + title, now it will be commit only.
Is it Ok?

Reply:
Nothing changes in that regard - I just moved the fmt that combined commit and title to another file.